src/System.IO.Compression/pkg/System.IO.Compression.pkgproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.0\System.IO.Compression.depproj" >
+      <SupportedFramework>net45;netcore45;wpa81</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\ref\System.IO.Compression.csproj">
+      <SupportedFramework>net46;netcore50;dnxcore50</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.IO.Compression.csproj">
+      <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
+    </ProjectReference>
+
+    <InboxOnTargetFramework Include="net45" />
+    <InboxOnTargetFramework Include="win8" />
+    <InboxOnTargetFramework Include="wpa81" />
+    <InboxOnTargetFramework Include="MonoAndroid10" />
+    <InboxOnTargetFramework Include="MonoTouch10" />
+    <InboxOnTargetFramework Include="xamarinios10" />
+    <InboxOnTargetFramework Include="xamarinmac20" />
+
+    <ProjectReference Include="win\System.IO.Compression.pkgproj" />
+    <ProjectReference Include="unix\System.IO.Compression.pkgproj"/>
+    <!-- unlike the windows implementation, the unix implementation p-invokes
```

Comment:
I think this comment is stale.  We used to have runtime dependencies here on the native package, but that was all moved to the runtime-specific packages.